magenta-lib/src/test/scala/magenta/deployment_type/ParamTest.scala

```diff
@@ -31,165 +42,100 @@ class ParamTest extends AnyFlatSpec with Matchers with MockitoSugar {
       actionNames = Seq("testAction")
     )
   )
-
-  "Param" should "register itself with a register" in {
-    implicit val register = new TestRegister
-
-    val param = Param[String]("test")
-
-    register.paramsList.size shouldBe 1
-    register.paramsList shouldBe Map("test" -> param)
-  }
-
-  it should "extract a value from a package using get" in {
-    implicit val register = new TestRegister
-    val pkg = DeploymentPackage(
+  def stubDeploymentPackage(pkgSpecificData: Map[String, JsValue]) =
```

Comment:
Introducing this `stubDeploymentPackage()` helper method has made each test case _much_ more concise - to the point where you can see what each test case is actually testing! 🎉